docs/source/whatsnew/1.0.0.rst

```diff
@@ -2,7 +2,7 @@
 
 .. py:currentmodule:: solarforecastarbiter
 
-1.0.0 (November ??, 2020)
+1.0.0 (November 17, 2020)
 -------------------------
 
 API Changes
```

Comment:
replace `-` with `~` in API changes section markup.

not sure why, but the `fill_persistence...` and `fill_prob...` links don't work.

double tick the ``gaps-only`` so it's rendered as code instead of emph